style.css

```diff
@@ -0,0 +1,51 @@
+body{
+	margin: 0;
+	padding: 0;
+}
+.menu a{
+	text-decoration: none;
+	font-family: arial;
+	color: #F0F9F9;
+	font-weight: bold;
+	text-transform: uppercase;
+	padding: 20px;
+}
+.menu{
+	float: right;
+	padding: 20px;
+}
+.open{
+	background-image: url("https://images.pexels.com/photos/956981/milky-way-starry-sky-night-sky-star-956981.jpeg?auto=compress&cs=tinysrgb&dpr=3&h=750&w=1260");
+	background-repeat: no-repeat;
+	background-size: cover;
+	min-height: 100vh;
+}
+h2, h3{
+	font-family: arial;
+	font-weight: bold;
+	color: #F0F9F9;
+	margin: 0;
+}
+.socials{
+	padding: 20px;
+	position: absolute;
+	top: 80vh;
+}
```

Comment:
So is the intent that the user never needs to scroll down on this homepage?
If so, you might be able to get that thing with the socials in the bottom corner working with some media queries. I think the format is
@media only screen and max-height {}
And have the socials resize. No way of figuring out whether they'll fit on the screen at various heights aside from manual testing though.